ios/FluentUI/Vnext/Shimmer/ShimmerTokens.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Shimmer style can be either concealing or revealing
+/// The style affects the default shimmer alpha value and the default shimmer tint color
+@objc public enum MSFShimmerStyle: Int, CaseIterable {
+    /// Concealing shimmer: the gradient conceals parts of the subviews as it moves leaving most parts of the subviews unblocked.
+    case concealing
+
+    /// Revealing shimmer: the gradient reveals parts of the subviews as it moves leaving most parts of the subview blocked.
+    case revealing
+}
+
+/// Design token set for the `Shimmer` control.
+open class ShimmerTokens: ControlTokens {
+    /// Determines whether the shimmer is a revealing shimmer or a concealing shimmer
+    public internal(set) var style: MSFShimmerStyle = .revealing
+
+    // MARK: - Design Tokens
+
+    /// The alpha value of the center of the gradient in the animation if shimmer is revealing shimmer
+    /// The alpha value of the view other than the gradient if shimmer is concealing shimmer
+    open var shimmerAlpha: CGFloat {
+        switch style {
+        case .concealing:
+            return 0
+        case .revealing:
+            return 0.4
+        }
+    }
+
+    /// Tint color of the view if shimmer is revealing shimmer
+    /// Tint color of the middle of the gradient if shimmer is concealing shimmer
+    open var tintColor: DynamicColor {
+        switch style {
+        case .concealing:
+            return DynamicColor(light: globalTokens.neutralColors[.white],
+                                dark: globalTokens.neutralColors[.grey8])
+        case .revealing:
+            return tint
+        }
+    }
+
+    open var darkGradient: ColorValue {
+        return globalTokens.neutralColors[.black]
+    }
+
+    open var tint: DynamicColor {
+        return DynamicColor(light: ColorValue(0xF1F1F1) /* gray50 */,
```

Comment:
this should be updated?

Reply:
I thought to use DynamicColor here so I could use the aliasTokens for at least that one color, thoughts?

Reply:
I think this code is right? @harrieshin what kind of update were you thinking?

Reply:
why are we using hex values at all? shouldn't we always be using the gray colors directly?